front/lib/plans/subscription.ts

```diff
@@ -318,9 +307,9 @@ export const pokeUpgradeWorkspaceToPlan = async (
     );
   }
 
-  if (newPlan.stripeProductId !== null) {
+  if (newPlan.code === PRO_PLAN_SEAT_29_CODE) {
```

Comment:
debatable. We decided to keep a "free upgraded" plan so IMO we should error when trying to upgrade someone to pro plan without a stripe sub.

Reply:
Agreed -> On Poké you should only be able to: 
1/ Downgrade to no plan
2/ Upgrade to a free plan
3/ Upgrade to an Enteprise plan.